packages/vapor/scss/components/navigation.scss

```diff
@@ -125,10 +125,20 @@
                             justify-content: space-between;
                             height: $navigation-menu-section-height;
                             margin-right: $navigation-right-margin;
+                            padding-right: 2px;
                             padding-left: $navigation-horizontal-space;
                             color: var(--navy-blue-80);
                             font-size: var(--default-font-size);
                             letter-spacing: 0.05em;
+                            border: 2px solid transparent;
+                            border-radius: 8px;
+
+                            &:focus-visible {
+                                background: var(--navy-blue-10);
+                                border: 2px solid var(--cerulean-blue-50);
```

Comment:
IMHO it looks better (more like the other focus styles that we already have on links for example) without those 2
```suggestion
```

Reply:
These colors match the new designs